crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -307,13 +335,13 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>>
     ) -> Result<Self, Self::Error> {
         let (payment_details, processing_options, subsequent_auth_information) =
             get_pm_and_subsequent_auth_detail(item)?;
-        let authorization_indicator_type =
-            item.router_data
-                .request
-                .capture_method
-                .map(|c| AuthorizationIndicator {
-                    authorization_indicator: c.into(),
-                });
+        let authorization_indicator_type = match item.router_data.request.capture_method {
```

Comment:
Why do we need this change?

Reply:
I refactored the CaptureMethod from `from` to `try_from` , in order to throw the error for `ManualMultiple` and `Scheduled`. this was a relative change for that